packages/@sanity/base/src/preview/availability.ts

```diff
@@ -59,8 +59,17 @@ function observeDocumentAvailability(id: string): Observable<DocumentAvailabilit
 }
 
 const fetchDocumentReadability = debounceCollect(function fetchDocumentReadability(
-  ids: string[]
+  args: string[][]
 ): Observable<DocumentAvailability[]> {
+  const uniqueIds = [...new Set(flatten(args))]
+  return from(chunk(uniqueIds, 300)).pipe(
+    mergeMap(fetchDocumentReadabilityChunked, 10),
```

Comment:
didn't know there was a second argument here. cool 😎

Reply:
It's the best second argument! <3